tests/ml_models/test_question_answering_pytest.py

```diff
@@ -0,0 +1,261 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+# How to run: pytest tests/ml_models/test_question_answering_pytest.py
+
+import json
+import os
+import shutil
+from zipfile import ZipFile
+
+import pytest
+
+from opensearch_py_ml.ml_models import QuestionAnsweringModel
+
+TEST_FOLDER = os.path.join(
+    os.path.dirname(os.path.abspath("__file__")), "tests", "test_model_files"
+)
+TESTDATA_FILENAME = os.path.join(
+    os.path.dirname(os.path.abspath("__file__")), "tests", "sample_zip.zip"
+)
+TESTDATA_UNZIP_FOLDER = os.path.join(
+    os.path.dirname(os.path.abspath("__file__")), "tests", "sample_zip"
+)
+
+default_model_id = "distilbert-base-cased-distilled-squad"
+
+def clean_test_folder(TEST_FOLDER):
```

Comment:
Can you please take a look at this [link](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) - It talks about how to use temporary files, directories with pytest. Do you think, using these fixtures help us?